components/form/builder/src/Icons/IconCloseTag.js

```diff
@@ -0,0 +1,17 @@
+import {memo} from 'react'
+import PropTypes from 'prop-types'
+
+const SVG = ({className}) => (
+  <svg viewBox="0 0 24 24">
+    <path
+      id="a"
```

Comment:
maybe we should use a more generic id, to avoid collisions with other ids

Reply:
you're right! that's what copying and pasting from one of our icons gets us haha good catch!

Reply:
and actually, I don't even think an `id` is necessary here